CondCore/Utilities/test/test_getPayloadData.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
+
+check_for_success() {
+    "${@}" && echo -e "\n ---> Passed test of '${@}'\n\n" || exit 1
+}
+
+check_for_failure() {
+    "${@}" && exit 1 || echo -e "\n ---> Passed test of '${@}'\n\n"
+}
+
+check_for_full(){
+    count=`echo ${@} | jq length`
```

Comment:
@smuzaffar it looks `jq` is not supported on the machines used for integration tests. Is there a good alternative to parse json files?

Reply:
@mmusich , you can do something like
```
count=`echo ${@} | python3 -c 'import json,sys;print(len(json.load(sys.stdin)))'
```

Reply:
thanks!